tests/unit/libutil/executable-path.cc

```diff
@@ -0,0 +1,29 @@
+#include <gtest/gtest.h>
+
+#include "executable-path.hh"
+
+namespace nix {
+
+#ifdef WIN32
+#  define PATH_VAR_SEP ";"
+#else
+#  define PATH_VAR_SEP ":"
+#endif
```

Comment:
Do we need to do this again, when `executable-path.cc` already has the (weirdly capitalized) `pATH_separator`?

Reply:
I did something similar for unit tests before, I don't mind having the unit tests have their own redundant one-off definition just for a bit of spec vs implementation separation purposes.